views/trials-details.html

```diff
@@ -78,5 +78,17 @@
       </dd>
     {% endfor %}
   </dl>
+
+  <h2>Publications</h2>
+  <ul>
+    {% for publication in trial.publications %}
+      <dt>
```

Comment:
@vitorbaptista 
We have many places with <dl> and <dd> in this view. Why we should use <ul> and <li> in this case only?  Should i change all <dl> and <dd> in this view?